web/styles/compose.css

```diff
@@ -188,19 +188,49 @@
         }
 
         & a.narrow_to_compose_recipients {
-            background: transparent;
-            font-size: 18px;
-            padding: 0 1px;
-            align-self: center;
+            display: flex;
+            align-items: center;
+            background: hsl(227deg 78% 59% / 12%);
```

Comment:
It'll probably help the redesign along if you make these color values into CSS variables.

Reply:
Added variables for all colors in both dark and light theme for the new button!